app/main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> str:
+    parts = command.split()
+
+    if not all([len(parts) == 3, parts[0] == "copy"]):
+        return (
+            f"Invalid command format. "
+            f"Use: copy '{parts[1]}' '{parts[2]}'."
+        ) if len(parts) > 2 else "Invalid command format."
```

Comment:
This part of your code is a bit confusing. It's not immediately clear what the conditions are for an invalid command. Consider using a more straightforward approach. For example, instead of using `all()`, you can use logical operators to check the conditions. Also, you should check that `parts[0]` is 'copy' before accessing `parts[1]` and `parts[2]` to avoid an `IndexError`. Here is a suggestion: 

python
if len(parts) != 3 or parts[0] != 'copy':
    return 'Invalid command format. Use: copy <source_file> <destination_file>.'
```